modules/ROOT/pages/keycloak/index.adoc

```diff
@@ -6,7 +6,16 @@ include::{partialsdir}/attributes.adoc[]
 [id='{context}_con_introduction-to-keycloak']
 = About {keycloak-service}
 
+The {keycloak-service} service allows you to add authentication and authorization to your mobile app.
+
+* Secure your mobile app using the industry standard OpenID Connect protocol
+* Add access control to your app based on user’s group membership
+* Easily implement SSO, multi factor authentication and Social Login support
+* Back end support for identity brokering and user federation
```

Comment:
I think Backend goes in 1 word?

Reply:
back-end is the correct term from IBM Style guide